### PR TITLE
[ANCHOR-142] Refactor sep24.interactive_url,sep24.more_info_url configuration and JWT schema

### PR DESCRIPTION
**Pull request by lijamie98**
<details>
  <summary>PR Checklist</summary>

### PR Structure

* [ ] This PR has reasonably narrow scope (if not, break it down into smaller PRs).
* [ ] This PR avoids mixing refactoring changes with feature changes (split into two PRs
  otherwise).
* [ ] This PR's title starts with name of package that is most changed in the PR, ex.
  `paymentservice.stellar`, or `all` or `doc` if the changes are broad or impact many
  packages.

### Thoroughness

* [ ] This PR adds tests for the most critical parts of the new functionality or fixes.
</details>

### What
1. Refactored sep24.interactive_url configuration
2. Refactored sep24.more_info_url configuration
3. Branch JWTToken to `Sep10Jwt`, `Sep24InteractiveUrlJwt`, and `Sep24MoreInfoUrlJwt`
4. Implemented the nested `Sep24InteractiveUrlJwt` and `Sep24MoreInfoUrlJwt` schemas.
5. Tests.

### Why

ANCHOR-142: Modify the JWT schema of the interactive URL

### Known limitations

N/A

### CODE REVIEW

File: platform/src/main/resources/config/anchor-config-default-values.yaml [hunk 1]
```diff
@@ -223,151 +199,120 @@ sep1:
...
-  ## the specified timeout period.
-  #
-  interactive_jwt_expiration: 3600
-
   ## @param: interactive_url
   # Configures the interactive URL where the platform server will redirect to start the SEP-24 interactive flow.
-  # If the type is `simple`, sep24.interactive_url.simple will be read to construct the url.
   #   base_uri: is the base_uri used to construct the interactive url
-  #   txn_fields: is the list of transaction fields that will be appended to the interactive url as query parameters
+  #   txn_fields: is an optional list of transaction fields that will be appended to the interactive url as query parameters. If empty, no fields from transaction will be appended.
```
Comment (JakeUrban) [inline platform/src/main/resources/config/anchor-config-default-values.yaml:282]: We'll make sure to add documentation about how these aren't necessarily the values stored in the AP, but can we add those comments here too?

**Comment by Ifropc** [comment]
I like separation of SEP-10 and SEP-24 JWTs. This makes it more clear in the code which JWT is being passed to methods

File: api-schema/src/main/java/org/stellar/anchor/api/callback/PutCustomerRequest.java [hunk 2]
```diff
@@ -67,6 +67,12 @@ public class PutCustomerRequest {
   @SerializedName("clabe_number")
   String clabeNumber;
 
+  @SerializedName("cbu_number")
+  String cbuNumber;
+
+  @SerializedName("cbu_alias")
+  String cbuAlias;
```
Comment (JakeUrban) [inline api-schema/src/main/java/org/stellar/anchor/api/callback/PutCustomerRequest.java:74]: Did I forget to add these fields in the 1.2.4 release? I see changes here I thought we already merged to develop?

**Review by JakeUrban — APPROVED** [review]

**Review by Ifropc — APPROVED** [review]

File: platform/src/main/resources/config/anchor-config-default-values.yaml (hunk 1 above)
Comment (lijamie98) [reply platform/src/main/resources/config/anchor-config-default-values.yaml:282]: Hey @JakeUrban, we only search for the transaction fields that are stored in the AP database. Can you give an example of retrieving the values not stored in AP?

File: api-schema/src/main/java/org/stellar/anchor/api/callback/PutCustomerRequest.java (hunk 2 above)
Comment (lijamie98) [reply api-schema/src/main/java/org/stellar/anchor/api/callback/PutCustomerRequest.java]: The PR is now rebased to the `develop` branch to include these fields.

**Pull request merged**